proto/desmos/profiles/v1beta1/genesis.proto

```diff
@@ -30,4 +30,12 @@ message GenesisState {
     (gogoproto.nullable) = false,
     (gogoproto.moretags) = "yaml:\"params\""
   ];
+
+  string port_id = 5 [
+    (gogoproto.moretags) = "yaml:\"port_id\"",
+    (gogoproto.customname) = "PortID"
+  ];
+
+  repeated desmos.profiles.v1beta1.Link links = 6
+      [ (gogoproto.moretags) = "yaml:\"links\"", (gogoproto.nullable) = false ];
```

Comment:
Since the links are also included inside each `Profile`, I don't think we need them here as well.

Reply:
It is for checking if link exists or not, used for preventing from double profiles problem.

Reply:
I think we should remove them. The double checking can still be done when initializing the data and storing them (eg. inside the keeper's method). Having a double list of links will result in duplicated code that needs to be written inside the genesis state

Reply:
Got it, will be store links here instead of using genesis file directly.
https://github.com/desmos-labs/desmos/blob/bd117008a64159184a57cdb9e43f986fec76c847/x/profiles/keeper/genesis.go#L27